src/main/xerus/monstercat/tabs/TabSound.kt

```diff
@@ -0,0 +1,65 @@
+package xerus.monstercat.tabs
+
+import javafx.application.Platform
+import javafx.scene.layout.HBox
+import javafx.scene.layout.VBox
+import javafx.scene.control.*
+import javafx.scene.media.AudioEqualizer
+import javafx.scene.media.EqualizerBand
+import javafx.geometry.Orientation
+import xerus.ktutil.javafx.*
+import xerus.ktutil.javafx.properties.*
+import xerus.monstercat.Settings
+import xerus.monstercat.api.Player
+
+class TabSound : VTab() {
```

Comment:
solange du nicht planst, hier noch was andres einzubauen, nenne es doch bitte TabEqualizer

Reply:
Prinzipiell ist der Equalizer so "speziell", dass er IMO keine einzelne Oberkategorie verdient. Falls später mal mehr Mischpult-Funktionalität kommen sollte (Balancing, Pitching, ...) spart man sich das Refactoring. (vermutlich ohnehin nicht mehr in diesem Projekt, je nach dem ob irgendwas von dem Code in deinen Musikplayer kommt)